modules.d/99base/dracut-lib.sh

```diff
@@ -1157,5 +1157,5 @@ remove_hostonly_files() {
 # returns OK if kernel_module is loaded
 # modprobe fails if /lib/modules is not available (--no-kernel use case)
 load_fstype() {
-    strstr "$(cat /proc/filesystems)" "$1" || modprobe "$1"
```

Comment:
Can you please elaborate on this change ? Why are you switching to grep as part of this PR ?

Reply:
Because I did so in the original in both places I noticed it and trunk moved one of them?

Reply:
I do not understand what "both places" and "trunk moved one of them" means. How is this particular change related to debashify ?

Reply:
"Both" means two of two, and "trunk moved one of them" means exactly the same: https://github.com/dracutdevs/dracut/commit/922c9e28ed87815cf6ae0b5ee911ff0ef616d1b0. In that case that was spelled with a ksh extension; consider this (in particular this hunk which is marked `refactor: strstr cat -> grep`) a free optimisation pass.

Reply:
> "Both" means two of two, and "trunk moved one of them" means exactly the same:
I genuinely did not understood which "two" you're referring to - you could have just listed a reference/link to the two places.  

Here is the link to this project code of conduct - https://github.com/dracutdevs/dracut/blob/master/docs/CODE_OF_CONDUCT.md